.github/workflows/nightly_tests.yml

```diff
@@ -180,6 +184,61 @@ jobs:
         pip install slack_sdk tabulate
         python utils/log_reports.py >> $GITHUB_STEP_SUMMARY
 
+  run_big_gpu_torch_tests:
+    name: Torch tests on big GPU (24GB)
+    strategy:
+      fail-fast: false
+      max-parallel: 8
```

Comment:
Would take this down to 2 so that we don't grab a bunch of big GPUs from the pool. Since we're running nightly and currently only a few models fall into this category it should be enough.